Quarto to Word Processor (R Studio & VS Code), and tutorials on R Markdown and Quarto from ds-econ.com

finnoh opened the pull request:
Add {quarto} to "{Word processor}" section.
Add {ds-econ tutorials} to {Tutorials} section.

<details>
  <summary>Short pitch</summary>

I added references to quarto: It is a open source framework following a similar style as R Markdown, however it can easily implement the execution of R, Python, Julia and OBJ. Furthermore, its writing is not limited to markdown files, as it can also render Jupyter Notebooks to scientific articles, presentations, and interactive websites. 

Besides adding quarto to this collection I added two tutorials (one for academic writing in R Markdown interconnected with Notion & Zotero, and one for setting up & writing in Quarto) from my own blog.

</details>

## Checklist

- [X] I have read and understood the [contribution guidelines](https://github.com/writing-resources/awesome-scientific-writing/blob/main/CONTRIBUTING.md).
- [X] If the entry is a software:
	- [X] it should be **maintained** (at least a commit / a release in the past 3 years),
	- [X] it is **open-source** with appropriate **license**
- [X] Table of contents has been updated (if a section is added / removed).
- [X] Contents have been sorted alphabetically.

<!-- NOTE: Please do not skip the template -->

maehr reviewed (changes requested):
Thank you very much for the contribution. Cuatro should only be added once and you may want to add a description.

finnoh commented:
Thank you for your feedback. I implemented those changes!

finnoh commented:
<img width="697" alt="image" src="https://user-images.githubusercontent.com/59664225/182014422-f4823704-a858-4385-a712-7b67b5a80f2e.png">

When using awesome-lint locally, I now only get these two errors. I believe these should not be a problem for the pull request? (I fixed a typo in the .README). Also, I removed a duplicate.
